xml/book_cloud_deploy.xml

```diff
@@ -38,6 +38,7 @@
   <xi:include href="depl_conf_admin_network.xml"/>
   <xi:include href="depl_conf_admin_crowbar.xml"/>
   <xi:include href="depl_crowbar_start.xml"/>
+  <xi:include href="depl_crowbar_extra-features.xml"/>
```

Comment:
A better name for this section may be "Customizing the crowbar lifecycle manager"